src/flow/src/transform.rs

```diff
@@ -179,6 +187,7 @@ mod test {
 
     pub fn create_test_ctx() -> FlownodeContext {
         let mut tri_map = IdToNameMap::new();
+        // deprecated: use `numbers_with_ts` instead since this table has no timestamp column
```

Comment:
Leave a fixme here @discord9 , don't forget to update the tests.

Reply:
> Leave a fixme here @discord9 , don't forget to update the tests.

will fix this in a separate PR once this get merged since it' unrelated to update datafusion